src/LibVLCSharp.WPF/ForegroundWindow.cs

```diff
@@ -106,8 +111,20 @@ void Background_Loaded(object sender, RoutedEventArgs e)
                 throw new VLCException("Unable to create WPF Window in VideoView.", ex);
             }
         }
-
         void Wndhost_LayoutUpdated(object? sender, EventArgs e)
+        {
```

Comment:
Why the code duplication? could it be the same method ? (with a correct name)

Reply:
Here you're right as well. Duplication not needed. Any suggestion for a good function name?

Reply:
- RefreshPosition
- MoveOverlayWindow
- AlignWindow

Pick you favorite or mix them. The function name should describe what it actually does